### PR TITLE
[MAINT] Add affiliations

### LINKED ISSUE #3808: Updating contributors affiliations
Closes #1511

## Goal

Use citation.cff file to document contributors:
- affiliations
- email
- ORCID

### Collect information about the contributors

"crowd source" information about contributors in this google spreadsheet:

https://docs.google.com/spreadsheets/d/1FN_u72YuwTeo-VuuCWp7SeKqseeUzoo-Sn0oAlNIZWw/edit#gid=2079625190

Nilearn contributors present at the OHBM 2023 hackathon can directly update their information in that spreadsheet during the hackthon.

For the other contributors, the goal will be to search and update manually the details for them.

### Update the citation.cff content

Depending on how many people decide to work on this, updating the citation.cff can be done:
- by writing a script to read from the table to update the authors section of the citation file
- manually updating the content of the citation.cff files in several pull requests:
  - Contributors from A to C
  - Contributors from D to J
  - Contributors from K to P
  - Contributors from Q to Z

### PR DESCRIPTION
**Pull request by Remi-Gau**
- Closes #3808 

Changes proposed in this pull request:

- get contributors infor from google spreadsheet set during OHBM hackathon
- get extra info from ORCID or github
- add affiliation to authors.rst

### CODE REVIEW

File: AUTHORS.rst [hunk 1]
```diff
@@ -48,145 +48,145 @@ Other contributors
...
+* `Alex Rothberg`_: Free Agency, New York, USA
+* `Alexandre Abadie`_: Inria, Paris
 * `Alexandre Abraham`_
-* `Alexandre Gramfort`_
+* `Alexandre Gramfort`_: Meta, France
 * `Alexandre Savio`_
-* `Alexis Thual`_
-* `Alisha Kodibagkar`_
+* `Alexis Thual`_: INRIA, Saclay, France
+* `Alisha Kodibagkar`_: MIT McGovern Institute, Cambridge, Massachusetts, United States
```
Comment (Remi-Gau) [inline AUTHORS.rst:59]: Could turn this into a table if we think it would look better.

**Comment by Remi-Gau** [comment]
If anyone feels like I can try to add map of the location of our contributors in the authors.rst file.

File: AUTHORS.rst [hunk 2]
```diff
@@ -48,145 +48,145 @@ Other contributors
 
 Some other past or present contributors are:
 
-* `Ahmad Chamma`_
+* `Ahmad Chamma`_: INRIA, Saclay, France
```
Comment (bthirion) [inline AUTHORS.rst:51]:
```suggestion
* `Ahmad Chamma`_: Inria, Saclay, France
```

File: AUTHORS.rst [hunk 3]
```diff
@@ -48,145 +48,145 @@ Other contributors
...
-* `Alexandre Abraham`_
-* `Alexandre Gramfort`_
+* `Alex Rothberg`_: Free Agency, New York, USA
+* `Alexandre Abadie`_: Inria, Paris
+* `Alexandre Abraham`_: Bazemont, Île-de-France, France
+* `Alexandre Gramfort`_: Meta, France
 * `Alexandre Savio`_
-* `Alexis Thual`_
-* `Alisha Kodibagkar`_
+* `Alexis Thual`_: INRIA, Saclay, France
```
Comment (bthirion) [inline AUTHORS.rst:58]:
```suggestion
* `Alexis Thual`_: Inria, Saclay, France
```

File: AUTHORS.rst [hunk 4]
```diff
@@ -48,145 +48,145 @@ Other contributors
...
+* `Anne-Sophie Kieslinger`_: Max Planck Institute for Human Cognitive and Brain Sciences, Leipzig, Germnay
+* `Ariel Rokem`_: University of Washington, Psychology, Seattle, Washington, 98107, USA
 * `Arthur Mensch`_
 * `Aswin Vijayan`_
-* `Audrey Duran`_
-* `Ben Cipollini`_
-* `Bertrand Thirion`_
+* `Audrey Duran`_: France
+* `Ben Cipollini`_: Facebook, Hayward, California, USA
+* `Bertrand Thirion`_: INRIA, Saclay, France
```
Comment (bthirion) [inline AUTHORS.rst:69]:
```suggestion
* `Bertrand Thirion`_: Inria, Saclay, France
```

File: AUTHORS.rst [hunk 5]
```diff
@@ -48,145 +48,145 @@ Other contributors
...
-* `Elizabeth DuPre`_
-* `Elvis Dohmatob`_
-* `Eric Larson`_
-* `Evan Edmond`_
-* `Fabian Pedregosa`_
+* `Dan Gale`_: Centre for Neuroscience Studie, Queen University, Kingston, ON, Canada
+* `Daniel Gomez`_: Cambridge, Massachusetts, USA
+* `Danilo Bzdok`_: McGill University & Mila Quebec AI Institute, Montréal, Canada
+* `David G Ellis`_: University of Nebraska Medical Center, Nebraska, USA
+* `Demian Wassermann`_: INRIA, Saclay, France
```
Comment (bthirion) [inline AUTHORS.rst:84]:
```suggestion
* `Demian Wassermann`_: Inria, Saclay, France
```

File: AUTHORS.rst [hunk 6]
```diff
@@ -48,145 +48,145 @@ Other contributors
...
-* `Greg Kiar`_
-* `Greydon Gilmore`_
-* `Guillaume Lemaitre`_
-* `Hao-Ting Wang`_
-* `Himanshu Aggarwal`_
+* `Gael Varoquaux`_: Inria, Saclay, France
+* `Gilles de Hollander`_: Vrije Universiteit, Amsterdam, The Netherlands
+* `Greg Kiar`_: Child Mind Institute
+* `Greydon Gilmore`_: Department of Mathematics, Western University, Toronto, Canada
+* `Guillaume Lemaitre`_: INRIA, Saclay, France
```
Comment (bthirion) [inline AUTHORS.rst:98]:
```suggestion
* `Guillaume Lemaitre`_: Inria, Saclay, France
```

File: AUTHORS.rst [hunk 7]
```diff
@@ -48,145 +48,145 @@ Other contributors
...
-* `Eric Larson`_
-* `Evan Edmond`_
-* `Fabian Pedregosa`_
+* `Dan Gale`_: Centre for Neuroscience Studie, Queen University, Kingston, ON, Canada
+* `Daniel Gomez`_: Cambridge, Massachusetts, USA
+* `Danilo Bzdok`_: McGill University & Mila Quebec AI Institute, Montréal, Canada
+* `David G Ellis`_: University of Nebraska Medical Center, Nebraska, USA
+* `Demian Wassermann`_: INRIA, Saclay, France
+* `Derek Pisner`_: St. Petersburg, FL
+* `Dimitri Papadopoulos Orfanos`_: NeuroSpin, C.E.A., Université Paris-Saclay, Gif-sur-Yvette, France
```
Comment (bthirion) [inline AUTHORS.rst:86]:
```suggestion
* `Dimitri Papadopoulos Orfanos`_: NeuroSpin, CEA, Université Paris-Saclay, Gif-sur-Yvette, France
```

File: AUTHORS.rst (hunk 3 above)
Comment (bthirion) [reply AUTHORS.rst:58]: Note that Alexis has multiple affiliations (INSERM, College de France)

File: AUTHORS.rst [hunk 8]
```diff
@@ -48,145 +48,145 @@ Other contributors
...
-* `Guillaume Lemaitre`_
-* `Hao-Ting Wang`_
-* `Himanshu Aggarwal`_
+* `Gael Varoquaux`_: Inria, Saclay, France
+* `Gilles de Hollander`_: Vrije Universiteit, Amsterdam, The Netherlands
+* `Greg Kiar`_: Child Mind Institute
+* `Greydon Gilmore`_: Department of Mathematics, Western University, Toronto, Canada
+* `Guillaume Lemaitre`_: INRIA, Saclay, France
+* `Hao-Ting Wang`_: Montréal Geriatrics Institute (CRIUGM), Montréal, Canada
+* `Himanshu Aggarwal`_: INRIA, Saclay, France
```
Comment (bthirion) [inline AUTHORS.rst:100]:
```suggestion
* `Himanshu Aggarwal`_: Inria, Saclay, France
```

File: AUTHORS.rst [hunk 9]
```diff
@@ -48,145 +48,145 @@ Other contributors
...
-* `Jon Haitz Legarreta Gorrono`_
+* `Jake Vogel`_: Philadelphia, USA
+* `Jan Margeta`_: KardioMe, Nova Dubnica, Slovakia
+* `Jaques Grobler`_: Anyline GmbH, Vienna, Austria
+* `Jason Gors`_: Insight Data Science, Mountain View, California, USA
+* `Javier Rasero`_: School of Data Science, University of Virginia, Viginia, USA
+* `Jean Kossaifi`_: Imperial College London, London, UK
+* `Jean-Rémi King`_: Meta AI, CNRS, Paris
+* `Jelle Roelof Dalenberg`_: University Medical Center Groningen, Groningen
+* `Jeremy Lefort-Besnard`_: INRIA Rennes, France
```
Comment (bthirion) [inline AUTHORS.rst:110]:
```suggestion
* `Jeremy Lefort-Besnard`_: Inria Rennes, France
```

File: AUTHORS.rst [hunk 10]
```diff
@@ -48,145 +48,145 @@ Other contributors
...
-* `Matthieu Joulot`_
+* `Martin Perez-Guevara`_: Criteo, France
+* `Martin Wegrzyn`_: Bielefeld University
+* `Mathias Goncalves`_: Stanford University, Stanford, California, USA
+* `Mathieu Dugré`_: Concordia University, Montréal, Canada
+* `Matthias Ekman`_: Donders Institute, Nijmegen, The Netherlands
+* `Matthieu Joulot`_: Aramis, Paris Brain Institute, Paris, France
 * `Maximilian Cosmo Sitter`_
-* `Mehdi Rahim`_
+* `Mehdi Rahim`_: Inria
```
Comment (bthirion) [inline AUTHORS.rst:138]: Currently with Air Liquide, France

File: AUTHORS.rst [hunk 11]
```diff
@@ -48,145 +48,145 @@ Other contributors
...
 * `Michael Eickenberg`_
```
Comment (bthirion) [inline AUTHORS.rst:139]: Flatiron Institute, NY, USA

File: AUTHORS.rst [hunk 12]
```diff
@@ -48,145 +48,145 @@ Other contributors
...
 * `Myeong Seop Song`_
-* `Natasha Clarke`_
-* `Neelay Shah`_
-* `Nicolas Gensollen`_
-* `Oliver Warrington`_
-* `Oscar Esteban`_
-* `Patrick Sadil`_
+* `Natasha Clarke`_: @SIMEXP, Montréal, Canada
+* `Neelay Shah`_: Max Planck Institute for Intelligent Systems, Germany
+* `Nicolas Gensollen`_: INRIA, Paris, France
```
Comment (bthirion) [inline AUTHORS.rst:149]:
```suggestion
* `Nicolas Gensollen`_: Inria, Paris, France
```

File: AUTHORS.rst [hunk 13]
```diff
@@ -48,145 +48,145 @@ Other contributors
...
+* `Peer Herholz`_: Origami lab, McGill University, Montréal, Canada
+* `Philippe Gervais`_: Zurich, Switzerland
+* `Pierre Bellec`_: Psychology Department, University of Montréal, Montréal, Canada
+* `Pierre Glaser`_: Gatsby Computational Neuroscience Unit, University College London, London, UK
+* `Pierre-Olivier Quirion`_: Canadian Center for Computational Genomics / Calcul, Québec, Canada
+* `Pradeep Reddy Raamana`_: University of Pittsburgh, Pittsburgh, PA, USA
 * `Rahul Brito`_
-* `Raphael Meudec`_
-* `Robert Luke`_
+* `Raphael Meudec`_: INRIA, Saclay, France
```
Comment (bthirion) [inline AUTHORS.rst:162]:
```suggestion
* `Raphael Meudec`_: Inria, Saclay, France
```

File: AUTHORS.rst [hunk 14]
```diff
@@ -48,145 +48,145 @@ Other contributors
...
+* `Tom Vanasse`_: Wisconsin Institute for Sleep and Consciousness, USA
 * `Vasco Diogo`_
-* `Vincent Michel`_
-* `Virgile Fritsch`_
-* `Yaroslav Halchenko`_
-* `Yasmin Mzayek`_
-* `Zvi Baratz`_
-* `Óscar Nájera`_
+* `Vincent Michel`_: http://www.logilab.fr/
+* `Virgile Fritsch`_: Inria - Parietal team, Saclay, France
```
Comment (bthirion) [inline AUTHORS.rst:185]: Now with EDF (but was with Inria Saclay when he contributed)

File: CITATION.cff [hunk 15]
```diff
@@ -30,48 +30,66 @@ authors:
   - given-names: Ahmad
     family-names: Chamma
     website: https://github.com/achamma723
+    affiliation: INRIA, Saclay, France
```
Comment (bthirion) [inline CITATION.cff:33]:
```suggestion
    affiliation: Inria, Saclay, France
```

File: CITATION.cff [hunk 16]
```diff
@@ -30,48 +30,66 @@ authors:
...
+    email: agramfort@meta.com
+    orcid: https://orcid.org/0000-0001-9791-4404
   - given-names: Alexandre
     family-names: Savio
     website: https://github.com/alex-savio
   - given-names: Alexis
     family-names: Thual
     website: https://github.com/alexisthual
-    affiliation: INRIA, France
+    affiliation: INRIA, Saclay, France
```
Comment (bthirion) [inline CITATION.cff:64]:
```suggestion
    affiliation: Inria, Saclay, France
```

File: CITATION.cff [hunk 17]
```diff
@@ -81,38 +99,53 @@ authors:
...
   - given-names: Ben
     family-names: Cipollini
     website: https://github.com/bcipolli
+    affiliation: Facebook, Hayward, California, USA
   - given-names: Bertrand
     family-names: Thirion
     website: https://pages.saclay.inria.fr/bertrand.thirion/
     email: bertrand.thirion@inria.fr
-    affiliation: INRIA, France
+    affiliation: INRIA, Saclay, France
```
Comment (bthirion) [inline CITATION.cff:111]:
```suggestion
    affiliation: Inria, Saclay, France
```

File: CITATION.cff [hunk 18]
```diff
@@ -122,21 +155,29 @@ authors:
...
     website: https://github.com/banilo
+    affiliation: McGill University & Mila Quebec AI Institute, Montréal, Canada
   - given-names: David G
     family-names: Ellis
     website: https://github.com/ellisdg
+    affiliation: University of Nebraska Medical Center, Nebraska, USA
   - given-names: Demian
     family-names: Wassermann
     website: https://github.com/demianw
+    affiliation: INRIA, Saclay, France
```
Comment (bthirion) [inline CITATION.cff:174]:
```suggestion
    affiliation: Inria, Saclay, France
```

File: CITATION.cff [hunk 19]
```diff
@@ -167,68 +214,101 @@ authors:
...
+    orcid: https://orcid.org/0000-0001-8915-496X
   - given-names: Greydon
     family-names: Gilmore
     website: https://www.greydongilmore.com/
+    affiliation: Department of Mathematics, Western University, Toronto, Canada
+    email: greydon.gilmore@gmail.com
   - given-names: Guillaume
     family-names: Lemaitre
     website: https://glemaitre.github.io/
+    affiliation: INRIA, Saclay, France
```
Comment (bthirion) [inline CITATION.cff:237]:
```suggestion
    affiliation: Inria, Saclay, France
```

File: CITATION.cff [hunk 20]
```diff
@@ -167,68 +214,101 @@ authors:
...
     family-names: Wang
     website: https://wanghaoting.com/
-    affiliation: Montreal Geriatrics Institute (CRIUGM), Montreal, Canada
+    affiliation: Montréal Geriatrics Institute (CRIUGM), Montréal, Canada
+    email: wang.hao-ting@criugm.qc.ca
+    orcid: https://orcid.org/0000-0003-4078-2038
   - given-names: Himanshu
     family-names: Aggarwal
     website: https://github.com/man-shu
+    affiliation: INRIA, Saclay, France
```
Comment (bthirion) [inline CITATION.cff:249]:
```suggestion
    affiliation: Inria, Saclay, France
```

File: CITATION.cff [hunk 21]
```diff
@@ -167,68 +214,101 @@ authors:
...
     website: https://github.com/kingjr
+    affiliation: Meta AI, CNRS, Paris
   - given-names: Jelle Roelof
     family-names: Dalenberg
     website: https://github.com/jrdalenberg
+    affiliation: University Medical Center Groningen, Groningen
   - given-names: Jeremy
     family-names: Lefort-Besnard
     website: https://jlefortbesnard.github.io/
+    affiliation: INRIA Rennes, France
```
Comment (bthirion) [inline CITATION.cff:293]:
```suggestion
    affiliation: Inria Rennes, France
```

File: CITATION.cff [hunk 22]
```diff
@@ -328,76 +441,101 @@ authors:
...
+    affiliation: University of Pittsburgh, Pittsburgh, PA, USA
+    email: raamanap@pitt.edu
+    orcid: https://orcid.org/0000-0003-4662-0558
   - given-names: Rahul
     family-names: Brito
     website: https://github.com/Rahul-Brito
   - given-names: Raphael
     family-names: Meudec
     website: https://github.com/RaphaelMeudec
+    affiliation: INRIA, Saclay, France
```
Comment (bthirion) [inline CITATION.cff:512]:
```suggestion
    affiliation: Inria, Saclay, France
```

File: CITATION.cff [hunk 23]
```diff
@@ -422,45 +562,70 @@ authors:
...
     website: https://tsalo.github.io/
     affiliation: Lifespan Informatics & Neuroimaging Center, University of Pennsylvania,
       USA
+    email: salot@pennmedicine.upenn.edu
+    orcid: https://orcid.org/0000-0001-9813-3167
   - given-names: Thomas
     family-names: Bazeille
     website: https://github.com/thomasbazeille
+    orcid: https://orcid.org/0000-0001-5377-9108
+    affiliation: INRIA, Parietal, Paris, France
```
Comment (bthirion) [inline CITATION.cff:594]:
```suggestion
    affiliation: Inria, Saclay, France
```

File: CITATION.cff [hunk 24]
```diff
@@ -422,45 +562,70 @@ authors:
...
     family-names: Diogo
     website: https://github.com/vascosa
   - given-names: Vincent
     family-names: Michel
     website: https://github.com/vmichel
+    affiliation: http://www.logilab.fr/
   - given-names: Virgile
     family-names: Fritsch
     website: https://github.com/VirgileFritsch
+    affiliation: Inria - Parietal team, Saclay, France
```
Comment (bthirion) [inline CITATION.cff:611]:
```suggestion
    affiliation: Inria, Saclay, France
```

File: CITATION.cff [hunk 25]
```diff
@@ -422,45 +562,70 @@ authors:
...
     family-names: Halchenko
     website: https://github.com/yarikoptic
+    affiliation: Dartmouth College, USA
+    email: debian@onerussian.com
+    orcid: https://orcid.org/0000-0003-3456-2493
   - given-names: Yasmin
     family-names: Mzayek
     website: https://github.com/ymzayek
-    affiliation: INRIA, France
+    affiliation: INRIA, Saclay, France
```
Comment (bthirion) [inline CITATION.cff:621]:
```suggestion
    affiliation: Inria, Saclay, France
```

**Review by bthirion — COMMENTED** [review]
Thx for doing that !

File: AUTHORS.rst (hunk 3 above)
Comment (Remi-Gau) [reply AUTHORS.rst:58]: yeah there are a few others like that: not sure how to handle it.

also not sure we should make any promise to keep all to those up to date: I feel that "affiliation at contribution time" feels more manageable for maintenance

though obviously if contributors want to drop by and update their affiliation they can do so

**Comment by Remi-Gau** [comment]
Will try to tackle the people with no affiliation or with just a country as affiliation but I think we should not aim to be 100% [completionist](https://www.dictionary.com/browse/completionist) on this one.

**Comment by Remi-Gau** [comment]
Will most likely move this to another PR but here is a little script that should help us consolidate the ORCID of some of our contributors

```python
"""list authors with no orcid id

ping the ORCID API to see what match we get
and only print when there are less than 5 results 
"""
import pandas as pd
import requests
from rich import print

from citation_cff_maint import read_citation_cff

citation = read_citation_cff()

author_list = []
orcid_list = []

for author in citation["authors"]:
    if "orcid" not in author:
        print(f"{author['given-names']} {author['family-names']}")

        url = f"https://pub.orcid.org/v3.0/search/?q=family-name:{author['family-names']}+AND+given-names:{author['given-names']}"

        response = requests.get(
            url,
            headers={
                "Accept": "application/json",
            },
        )

        if response.status_code == 200:
            resp = response.json()
            if resp["num-found"] > 0 and resp["num-found"] < 6:
                author_list.append(
                    f"{author['given-names']} {author['family-names']}"
                )
                orcid = [x["orcid-identifier"]["uri"] for x in resp["result"]]
                orcid_list.append(", ".join(orcid))

df = pd.DataFrame({"author": author_list, "orcid": orcid_list})
df.to_csv("orcid.tsv", index=False, sep="\t")
```


author	orcid
Aina Frau-Pascual	https://orcid.org/0000-0001-6311-8426
Alexandre Abraham	https://orcid.org/0000-0003-3693-0560
Alexandre Savio	https://orcid.org/0000-0002-6608-6885, https://orcid.org/0000-0002-6067-6606
Anne-Sophie Kieslinger	https://orcid.org/0000-0001-9240-4077
Arthur Mensch	https://orcid.org/0000-0001-7866-0461
Aswin Vijayan	https://orcid.org/0000-0002-0564-0774, https://orcid.org/0000-0002-5039-957X, https://orcid.org/0000-0002-1905-4194
Audrey Duran	https://orcid.org/0000-0002-1037-5098
Caglar Cakan	https://orcid.org/0000-0002-1902-5393
Christian Gerloff	https://orcid.org/0000-0003-2718-2089, https://orcid.org/0000-0002-6484-8882
Connor Lane	https://orcid.org/0009-0009-0164-5672, https://orcid.org/0000-0002-6456-6308, https://orcid.org/0009-0003-6723-8013, https://orcid.org/0000-0002-5527-4768
Céline Delettre	https://orcid.org/0000-0001-6568-0295
Danilo Bzdok	https://orcid.org/0000-0003-3466-6620
Derek Pisner	https://orcid.org/0000-0002-1228-0201
Elvis Dohmatob	https://orcid.org/0000-0003-1421-6972
Evan Edmond	https://orcid.org/0000-0003-4641-3583
Fabian Pedregosa	https://orcid.org/0000-0003-4025-3953
François Paugam	https://orcid.org/0000-0002-8161-7699
Ian Abenes	https://orcid.org/0000-0002-7625-1557
Jake Vogel	https://orcid.org/0000-0002-9936-6832
Javier Rasero	https://orcid.org/0000-0003-1661-2856
Jean Kossaifi	https://orcid.org/0000-0002-4445-3429
Jean-Rémi King	https://orcid.org/0000-0002-2121-170X, https://orcid.org/0000-0003-0061-0872, https://orcid.org/0000-0001-9572-8210, https://orcid.org/0000-0001-5759-7985
Jelle Roelof Dalenberg	https://orcid.org/0000-0001-8580-5358
Jeremy Lefort-Besnard	https://orcid.org/0000-0002-8033-4953, https://orcid.org/0000-0001-7127-7558
Johannes Wiesner	https://orcid.org/0000-0001-7040-3516
Jona Sassenhagen	https://orcid.org/0000-0002-9935-8621
Joshua Teves	https://orcid.org/0000-0002-7767-0067
Julio A Peraza	https://orcid.org/0000-0003-3816-5903
Koen Helwegen	https://orcid.org/0000-0002-1247-6626
Konrad Wagstyl	https://orcid.org/0000-0003-3439-5808
Konstantin Shmelkov	https://orcid.org/0000-0002-1555-1126
Leonard Sasse	https://orcid.org/0000-0002-2400-3404
Loic Tetrel	https://orcid.org/0000-0002-1501-331X
Martin Wegrzyn	https://orcid.org/0000-0001-8025-4075
Matthias Ekman	https://orcid.org/0000-0003-1254-1392
Mehdi Rahim	https://orcid.org/0000-0002-9570-4144
Michael Eickenberg	https://orcid.org/0000-0003-0656-4803
Michael Notter	https://orcid.org/0000-0002-5866-047X
Michael Waskom	https://orcid.org/0000-0002-9817-6869
Moritz Boos	https://orcid.org/0000-0002-1219-4901
Natasha Clarke	https://orcid.org/0000-0002-6547-8609, https://orcid.org/0000-0003-2375-4510, https://orcid.org/0000-0003-2455-3614, https://orcid.org/0000-0002-4508-6851
Neelay Shah	https://orcid.org/0000-0001-7950-1956, https://orcid.org/0000-0003-3667-4114
Nicolas Gensollen	https://orcid.org/0000-0001-7199-9753
Oliver Warrington	https://orcid.org/0000-0002-3592-7071
Patrick Sadil	https://orcid.org/0000-0003-4141-1343
Paul Bogdan	https://orcid.org/0000-0002-4362-6084, https://orcid.org/0000-0003-2118-0816
Philippe Gervais	https://orcid.org/0000-0002-2656-3931, https://orcid.org/0000-0003-2172-879X, https://orcid.org/0000-0003-1622-6066
Pierre Glaser	https://orcid.org/0000-0003-4146-973X
Rahul Brito	https://orcid.org/0000-0003-0343-0455
Raphael Meudec	https://orcid.org/0000-0001-9970-5745
Robert Luke	https://orcid.org/0000-0002-4930-8351
Roberto Guidotti	https://orcid.org/0000-0001-6487-3968, https://orcid.org/0000-0002-0807-6005
Ronald Phlypo	https://orcid.org/0000-0003-4310-7994
Ryan Hammonds	https://orcid.org/0000-0002-5032-3241
Sachin Patalasingh	https://orcid.org/0000-0001-5627-2249
Sage Hahn	https://orcid.org/0000-0003-3560-2952
Simon Steinkamp	https://orcid.org/0000-0002-6437-0700
Vasco Diogo	https://orcid.org/0000-0002-7965-9013, https://orcid.org/0000-0001-5732-9390, https://orcid.org/0000-0003-1793-4443
Yasmin Mzayek	https://orcid.org/0000-0003-2904-2530

**Comment by Remi-Gau** [comment]
Will merge this one and move the ORCID update to another issue.

**Comment by Remi-Gau** [comment]
For the record.

> Note that Alexis has multiple affiliations (INSERM, College de France)

Citation.cff do not yet support multiple affiliations but this is the discussion thread about it.

https://github.com/citation-file-format/citation-file-format/issues/268

**Pull request merged**